lib/ansible/modules/net_tools/basics/read_mail.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Caio Ramos <caioramos97@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: read_mail
+
+short_description: Module for reading emails from a IMAP server
+
+version_added: "2.9"
+
+description:
+    - Get emails from a IMAP server using basic filtering rules and returns
+      the email count and a list of dictionaries where each represents an email
+      that satisfied the filters. This module is intended to be used by other
+      tasks that take actions depending on the result of the emails.
+
+options:
+    host:
+        description:
+            - IMAP server host
+        required: false
+        default: localhost
+    port:
+        description:
+            - IMAP server port (defaults to 993, if TLS is chosen, or 143 if not)
+        required: false
+        default: -1
```

Comment:
I'm not sure about the syntax in ansible documentation, but as we have default value, we need to document this field